src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -86,29 +85,86 @@ def __init__(
         >>> pd.Index(list('abc'))
         Index(['a', 'b', 'c'], dtype='object')
 
+        # Snowpark pandas casts all int types to int64 by default
```

Comment:
```suggestion
        # Snowpark pandas only supports signed integers so cast to uint won't work
```
whether it is int64 or int8 is decided by the SQL engine. Can you update all those comments?

Reply:
Yep I updated this in the most recent push. Should I also mention "whether it is int64 or int8 is decided by the SQL engine" ?